conformance/utils/suite/suite.go

```diff
@@ -234,6 +236,7 @@ func NewConformanceTestSuite(options ConformanceOptions) (*ConformanceTestSuite,
 
 	suite := &ConformanceTestSuite{
 		Client:           options.Client,
```

Comment:
Is there much point allowing someone to pass in a `Client` anymore if its going to be ignored anyways?

Reply:
This is a fair comment. It should be removed

Reply:
Would a follow up PR be suitable?